test/test.spec.ts

```diff
@@ -490,7 +517,7 @@ describe('service', function () {
 describe('embed', function () {
   let powerbi: service.Service;
   let $element: JQuery;
-  let $container: JQuery;
+  let $reportEmbedContainer: JQuery;
```

Comment:
Why rename this?  I think it is supposed to represent an element that contains an embed.  The embed could be report or dashboard.